src/shapes/IText/IText.ts

```diff
@@ -149,6 +150,8 @@ export class IText<
    */
   declare caching: boolean;
 
+  static textLayoutProperties = Text.textLayoutProperties;
```

Comment:
no need for this
static props are inherited

Reply:
I see #8662

Reply:
are they? my understand is that they are inherited only in static methods.
If is not needed even better.

Reply:
![image](https://user-images.githubusercontent.com/34343793/216825446-75b7a06d-f593-4122-ac5e-23e4eada65ad.png)

```js
class A {
 static m=5
}
class B extends A {}
console.log(B.m)
```